src/cli.ts

```diff
@@ -28,7 +28,15 @@ export class CLI {
         let mvnCommand = this.getCommand(MAVEN_COMMAND);
         let gradleCommand = this.getGradleCommand();
         if (!gaugeCommand || gaugeCommand === '') return new CLI(gaugeCommand, {}, mvnCommand, gradleCommand);
-        let gv = spawnSync(gaugeCommand, [GaugeCommands.Version, GaugeCommands.MachineReadable]);
+        let options: CommonSpawnOptions = {};
+        if(platform() === "win32") {
```

Comment:
I remember making a similar change for gauge typescript at

https://github.com/getgauge/gauge-ts/blob/a34fb7254a65cd0a9dff11c8dbac56ac755b0448/gauge-ts/launcher.mjs#L40

Instead of checking for win32 can't we just pass the `shell : true` option for all platforms? 🤔

Reply:
I checked the `shell: true` options in the [official doc](https://nodejs.org/docs/latest/api/child_process.html#shell-requirements), is there any linux shell that does not have '-c' option? And in [another question from stackexchange](https://unix.stackexchange.com/questions/739837/on-what-occasions-should-you-use-sh-c-instead-of-directly-executing-a-program), it seems we should avoid using shell option.